yarn.lock

```diff
@@ -1,13013 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please keep yarn.lock in the history. It makes sure everyone is running the same versions of all the dependencies.

Reply:
yarn.lock added back in